mysite/blog/templates/blog/post/detail.html

```diff
@@ -10,4 +10,28 @@ <h1>{{ post.title }}</h1>
     <p>
         <a href="{% url 'blog:post_share' post.id %}">Share this post</a>
     </p>
+    {% with comments.count as total_comments %}
+        <h2>{{ total_comments }} comment{{ total_comments|pluralize }}</h2>
+    {% endwith %}
+    {% for comment in comments %}
+        <div class="comment">
+            <p class="info">
+                Comment {{ forloop.counter }} by {{ comment.name }}
```

Comment:
Need explanation

Reply:
- {% for %} template tag to loop through comments

- {{ forloop.counter}} variable, which contains the loop counter in each iteration. Then, display the name of the user who posted the comment.